src/EPR.RegulatorService.Facade.API/Controllers/InviteRegulatorController.cs

```diff
@@ -38,8 +43,9 @@ public async Task<IActionResult> CreateInviteEnrollment([FromBody] RegulatorInvi
                 }
 
                 var userId = User.UserId();
-
-                if (userId == default)
+                Guid validUserId;
+                
+                if ((!Guid.TryParse(userId.ToString(), out validUserId)) || validUserId == Guid.Empty)
```

Comment:
as there are so many changed related to this guid check it might be cleaner to extract an extension method?

Reply:
Done.